R/data.R

```diff
@@ -48,16 +48,16 @@
 
 #' Midwest demographics
 #'
-#' Demographic information of midwest counties
+#' Demographic information of midwest counties from 2000 US census
 #'
 #' @format A data frame with 437 rows and 28 variables:
 #' \describe{
-#'  \item{PID}{}
-#'  \item{county}{}
-#'  \item{state}{}
-#'  \item{area}{}
-#'  \item{poptotal}{Total population}
-#'  \item{popdensity}{Population density}
+#'  \item{PID}{Unique county identifier.}
+#'  \item{county}{County name.}
+#'  \item{state}{State to which county belongs to.}
+#'  \item{area}{Area of county (units unknown).}
```

Comment:
I tried looking up the land area units for this and nothing came out as something reasonable. So I've just made this `units unknown`